### PR TITLE
Fix out-of-bound issue in call to LUT in GEM-CSC trigger

### PR DESCRIPTION
**Pull request by dildick**
#### PR description:

This PR should fix a bug reported in the GEM-CSC trigger in CMSSW_11_0_X samples, when a pad number goes out of bounds in the `[]` operator. I also replaced `[]` with `at()` and added a few assert statements. The bug was introduced when the pad numbering in GE1/1 was changed from 1->192 to 0->191, and in GE2/1 from 1->384 to 0->383.

#### PR validation:

Tested on 4000 events in Run-3 sample `/DYToLL_M-50_TuneCP5_14TeV-pythia8/Run3Winter20DRMiniAOD-DRFlatPU30to80_110X_mcRun3_2021_realistic_v6-v2/GEN-SIM-RAW`. Also tested with WF 23234.0

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

N/A

FYI @eyigitba @tahuang1991

### CODE REVIEW

File: L1Trigger/CSCTriggerPrimitives/src/CSCGEMMotherboard.cc [hunk 1]
```diff
@@ -240,8 +253,8 @@ int CSCGEMMotherboard::getRoll(const GEMPadDigiId& p) const { return GEMDetId(p.
 int CSCGEMMotherboard::getRoll(const GEMCoPadDigiId& p) const { return p.second.roll(); }
 
 std::pair<int, int> CSCGEMMotherboard::getRolls(const CSCALCTDigi& alct) const {
-  return std::make_pair((getLUT()->get_csc_wg_to_gem_roll(theParity))[alct.getKeyWG()].first,
-                        (getLUT()->get_csc_wg_to_gem_roll(theParity))[alct.getKeyWG()].second);
+  const auto& mymap(getLUT()->get_csc_wg_to_gem_roll(theParity));
+  return std::make_pair(mymap.at(alct.getKeyWG()).first, mymap.at(alct.getKeyWG()).second);
```
Comment (fwyzard) [inline L1Trigger/CSCTriggerPrimitives/src/CSCGEMMotherboard.cc:257]: isn't this the same as
```suggestion
  return mymap.at(alct.getKeyWG());
```
?

**Comment by silviodonato** [comment]
please test

**Comment by silviodonato** [comment]
urgent

**Comment by silviodonato** [comment]
merge

**Pull request merged**